tests/testthat/indention_token_dependent/token_dependent_complex-out.R

```diff
@@ -2,30 +2,39 @@ call(call(
   2
 ))
 
-call(call(1,
-          2))
+call(call(
+  1,
+  2
+))
 # multi-line: no indention based on first vall
 call(a(b(c({
 }))))
 
+call(
+  call(
+    2
+  ),
+  5
+)
+
+
 call(call(
-  2),
-     5)
-
-
-call(call(1,
-          2, c(
-            3
-        )))
-
-call(1,
-     call2(3, 4, call(3,
-                      4, call(5, 6, call(
-                        2
-                      )
-                      )
-     )
-     )
+  1,
+  2, c(
```

Comment:
We seem to need a rule that requires a newline before multi-line function arguments. Do we have a self-contained writeup for the new function call rules? Maybe open a PR to the tidystyle repo so that it's not lost?